swc/private/swc.bzl

```diff
@@ -175,6 +180,30 @@ def _impl(ctx):
     args.add_all(ctx.attr.args)
     args.add_all(["--source-maps", ctx.attr.source_maps])
 
+    plugin_cache = []
+    plugin_args = []
+    if ctx.attr.plugins:
+        plugin_cache = [ctx.actions.declare_directory("{}_plugin_cache".format(ctx.label.name))]
```

Comment:
very creative :)

Reply:
ha, I was pretty proud of myself when I came up with that one